spec/acceptance/golden_masters_spec.rb

```diff
@@ -8,13 +8,13 @@
   it 'returns the correct results for the old spacewarps data' do
     storage = RetirementSwap::Storage::Memory.new
     output = RetirementSwap::Output::IOWriter.new(StringIO.new)
-    retirement_swap = RetirementSwap::SwapAlgorithm.new(storage, output)
+    processor = RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"})
```

Comment:
Line is too long. [117/80]<br>Space inside { missing.<br>Space inside } missing.